### PR TITLE
Add the benchmark test of Insertionsort

### PR DESCRIPTION
**Pull request by Sunbelife**
I added the benchmark test of Insertionsort

Signed-off-by: Sunbelife Sunbelife@foxmail.com

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/Llama-Group/Project-Llama/pull/52" target="_blank">ReviewNinja

File: llama_cpp/benchmark/sort_benchmark/sort_benchmark.cpp [hunk 1]
```diff
@@ -17,6 +18,7 @@ static void StdSortString(benchmark::State& state) {
     }
 }
 
+//BFSort
```
Comment (0xsuu) [inline llama_cpp/benchmark/sort_benchmark/sort_benchmark.cpp:21]: Add space after //

File: llama_cpp/benchmark/sort_benchmark/sort_benchmark.cpp [hunk 2]
```diff
@@ -41,6 +43,31 @@ static void BFSortDouble(benchmark::State& state) {
     }
 }
 
+//InSort
```
Comment (0xsuu) [inline llama_cpp/benchmark/sort_benchmark/sort_benchmark.cpp:46]: Add space after //

**Comment by 0xsuu** [comment]
Fix the push build fail. @Sunbelife

**Comment by 0xsuu** [comment]
OK to merge

**Pull request merged**